src/sentry/api/endpoints/check_am2_compatibility.py

```diff
@@ -19,7 +19,7 @@
 class CheckAM2CompatibilityEndpoint(Endpoint):
```

Comment:
This one really should be in getsentry. Synced with @vbro and the plan is to move it to private for now before that change happens.